contracts/extensions/StreamingPayments.sol

```diff
@@ -420,6 +506,34 @@ contract StreamingPayments is ColonyExtensionMeta {
     return min(domainBalance, _amountEntitledToClaimNow);
   }
 
+  function getAmountClaimableInTime(
+    uint256 _id,
+    uint256 _from,
+    uint256 _until
+  ) private view returns (uint256) {
+    StreamingPayment storage streamingPayment = streamingPayments[_id];
+
+    if (_from >= _until) {
```

Comment:
Why not `require(_from < _until)`?

Reply:
In general, if able to return a valid value from such a function, I think it's generally more correct to do so, rather than be opinionated about e.g. 'zero is an error', and let functions using it make that decision.

In this case specifically, if `getAmountClaimableInTime` errored when it would return 0, then you wouldn't be able to e.g. `setStartTime` if you had created the stream in a way that had a lifetime payout of zero.

Reply:
I hear that, although for the record we often treat 0 values as errors elsewhere in the codebase.

Reply:
What I said above is true, but I misunderstood what you were actually suggesting here, which is only requiring that `require(_from < _until)` and still returning 0 in other, 'healthy' states. 

We could have the require, but we'd still need a special case in the event of equality... if you feel strongly about it, we can put in the require, but I'm still happy with this as-is.

> I hear that, although for the record we often treat 0 values as errors elsewhere in the codebase.

Your statement is true, but I'm not sure this is a great argument... are you suggesting that, for consistency, `balance(address)` on our token should error if the address in question has no tokens?

Reply:
My preference would be for the opposite, to allow for 0 arguments wherever they are no explicitly invalid values.